examples/starwars/droid.bal

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) 2022 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import starwars.datasource as ds;
+
+# An autonomous mechanical character in the Star Wars universe
+distinct isolated service class Droid {
+
+    private final readonly & ds:DroidRecord droid;
+
+    function init(ds:DroidRecord droid) {
+        self.droid = droid.cloneReadOnly();
+    }
+
+    # The unique identifier of the droid
+    # + return - The id
+    resource function get id () returns string {
```

Comment:
There shouldn't be a space after the resource path. Please check all the places.
```suggestion
    resource function get id() returns string {
```